extensions/amp-story/0.1/amp-story.css

```diff
@@ -197,6 +197,22 @@ amp-story:not([desktop]) >
   transform: translateY(1000%) !important;
 }
 
+amp-story:not([desktop]) > amp-story-page[distance=0] {
+  transform: translateY(0) !important;
+}
+
+amp-story:not([desktop]) > amp-story-page[distance=1] {
+  transform: translateY(100%) !important;
+}
+
+amp-story:not([desktop]) > amp-story-page[distance=2] {
+  transform: translateY(200%) !important;
+}
+
+amp-story:not([desktop]) > amp-story-page[distance=3] {
+  transform: translateY(300%) !important;
+}
```

Comment:
How are pages at distance >= 4 handled? Can we use CSS vars and `calc()` here instead?

Reply:
Shouldn't be a problem, since the default styling for `amp-story-page`will put all pages with distance >= 4 over the limit for runtime scheduling.

Reply:
I'm also wary of using CSS variables due to [support.](https://caniuse.com/#feat=css-variables)

Reply:
Alright, mind adding a comment here explaining that?

Reply:
Good idea, done!